modules/Microsoft.DBforMySQL/flexibleServers/.test/.scripts/Get-PairedRegion.ps1

```diff
@@ -0,0 +1,16 @@
+﻿param(
+    [string] $Location
+)
+
+# Sleep for role assignment propagation
+Start-Sleep -Seconds 10
+
+$PairedRegionName = Get-AzLocation |
+    Where-Object -FilterScript { $PSItem.Location -eq $Location -or $PSItem.DisplayName -eq $Location } |
```

Comment:
```suggestion
    Where-Object -FilterScript { $Location - in @($PSItem.Location, $PSItem.DisplayName) } |
```

Reply:
@AlexanderSehr will commit but then keep the parameter names `FilterScript` as its best practice for readability